app/views/layouts/_proxy_notice.html.erb

```diff
@@ -0,0 +1,18 @@
+<div id="proxy-notice">
+  <div class="content">
+    <p class="english">Notice: (1) You are using a proxy site that is not affiliated with AO3. (2) The entity that set up the proxy site can see whatever data you submit to the AO3 through the proxy site, including your IP address and the content of comments.</p>
+    <p class="chinese"> 重要提示：<br/>
```

Comment:
Do we plan to selectively show the notice in different languages, hence the "english" and "chinese" classes?

Reply:
No, I just thought it might be nice to have the class names there.